src/common/components/Sidebar.js

```diff
@@ -63,6 +63,7 @@ function Sidebar({ onClick }) {
   const navigate = useNavigate();
   const isLoggedIn = useSelector((state) => state.login.isLoggedIn);
   const dispatch = useDispatch();
+  const loginedUserId = useSelector((state) => state.login.userId);
```

Comment:
```suggestion
  const loggedInUserId = useSelector((state) => state.login.userId);
```
login의 과거형이 logged In 이기 때문에  loggedInUserId 가 변수명으로 더 적절할 것 같아요. .